packages/rocketchat-blockstack/server/main.js

```diff
@@ -0,0 +1,7 @@
+import { Meteor } from 'meteor/meteor';
+
+import './routes.js';
+import './settings.js';
+import './loginHandler.js';
+
+Meteor.isDevelopment = (process.env.ROOT_URL.indexOf('localhost') !== -1);
```

Comment:
this is not being used by this PR

Reply:
actually, this is [set by default](https://docs.meteor.com/api/core.html#Meteor-isDevelopment).. is there a reason to override the default value/logic?

Reply:
There isn't, was a trash from the original PR